### DISCUSSION TITLE
چند سوال در مورد پنل

### DISCUSSION THREAD

**Discussion by lostsoul6** [hiddify/Hiddify-Manager#423] [Q&A سوالات رایج]
سلام عزیزان وقت بخیر . چند تا سوال داشتم ممنون میشم عزیزان پاسخ بدن

1 - من دنبال پنلی هستم که بشه داخلش node های مختلف اضافه کرد یعنی سرورهای مختلف و کشورهای مختلف داشته باشه بعد کاربر بتونه هر کدوم از سرورها رو خواست لینکش رو بگیره و باهاش کار کنه و حجمش به درستی کم بشه . آیا هیدیفای همچین قابلیتی داره ؟

2 - بنده چند روز وقت گذاشتم تا پنل v2board رو نصب کنم ولی بعدش متوجه شدم اصلا vless ساپورت نمیکنه و فقط vmess داره ! خواستم بدونم پنل هیدیفای همه پروتکل هایی که توی x-ui هست رو داره ؟ مهم ها vmess و vless هستند

3 - وقتی داخل x-ui کانفیگ میسازی خیلی گزینه ها هست مثل transmission که میزنیم ws یا grpc مثلا یا گزینه های مربوط به TLS و غیره . آیا هیدیفای همه اینارو داره ؟

4 - مصرف منابع هیدیفای چقدر هست و چند کاربر جواب میده ؟ آیا بهینه هست که به مشکل نخوره توی اتصال بالا؟

5 - آیا پنل خارجی یا هر پنل معتبر دیگه ای هست که خوب باشه و کارایی که گفتم مثل افزودن node یا ساپورت vless و غیره رو داشته باشه ؟ اگر بله ممنون میشم معرفی کنید .

تشکر .

**Comment by zZedix** [comment]
۱ - هیدیفای این امکان رو داره ولی هنوز منتشر نشده
۲ - هیدیفای از تمامی پروتکل های رایج XRAY پشتیبانی میکنه یعنی Vless , Vmess  , Trojan , SS و .....
۳ - داخل هیدیفای هم انواع کانفیگ به صورت پیش فرض وجود داره هم میتونید به دلخواه تنظیمات مورد نظر و انجام بدید
۴ - من روی یه سرور ۱ core و 2GB Ram نصب دارم و هیچ مشکلی ندارم 
۵ - بله داره ! XMPLUS یا Proxypanel

**Reply by lostsoul6** [reply]
> ۱ - هیدیفای این امکان رو داره ولی هنوز منتشر نشده ۲ - هیدیفای از تمامی پروتکل های رایج XRAY پشتیبانی میکنه یعنی Vless , Vmess , Trojan , SS و ..... ۳ - داخل هیدیفای هم انواع کانفیگ به صورت پیش فرض وجود داره هم میتونید به دلخواه تنظیمات مورد نظر و انجام بدید ۴ - من روی یه سرور ۱ core و 2GB Ram نصب دارم و هیچ مشکلی ندارم ۵ - بله داره ! XMPLUS یا Proxypanel

ممنون بابت راهنمایی شما عزیز . کاملترین پنلی که برای این کار وجود داره از بین hiddify و xmplus و proxypanel کدوم هست ؟ حتی اگر پولی هم باشه مشکلی نیست فقط چیزایی که بالا گفتم رو ساپورت کنه

بعد اون قابلیت سوال اول که گفتید هنوز منتشر نشده کی منتشر میشه نسخه درست و تست شده اش؟

**Reply by zZedix** [reply]
به نظر من Hiddify ! چون نصبش بی نهایت راحته و سرعت آپدیت و بروزرسانی امکانات بسیار بالاست ! علاوه بر اون این سیستم یه جورایی بر اساس نیاز های داخل ساخته شده ! یک Community بسیار خوب داره و ادمین در مورد پشتیبانی سنگ تموم میذاره ! امیدواریم که بزودی پنل مادر هم رو نمایی بشه ! hiddify بسیارکامل و User friendly هست
